.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
+
+._.DS_Store
 .idea
-package-lock.json
-build
\ No newline at end of file
+package-lock.json
```

Comment:
It's better to keep package-lock.json in repository, it helps other developers to install same version of any subdependency. example: if you will have package A in your package.json with explicit version, for example 1.0, and it will have it's own dependency, B, with version marked with star "*", which can be resolved to version 0.1.0. If you wont use package-lock file, you will resolve dependency B to 0.1.0, but if I install your packages after few months, dependency B can be updated to that time and resolved to 2.1.0, which can be not compatible with your code. So you will have same package.json as i am, but code on your machine will work, but wont work on my. package-lock file remember exact version of every dependency, so if you will keep it in repo, i will install exactly same version of all files as you, so it will prevent some tricky bugs with dependencies.